### DISCUSSION TITLE
Rich text (markdown)

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#6911] [Help] [answered]
Is there any way to pre-select the "Toggle preview" button?
![image](https://user-images.githubusercontent.com/95284278/181480179-4e5f2534-0d97-40ce-8fbd-81b3bc4b7ec3.png)

Thanks!

**Comment by melohagan** [comment, marked as answer]
Hey @kasgithub8 

There is no way to currently do this that I can see, but it's a good idea! 
I've raised a feature request here: https://github.com/Budibase/budibase/issues/6979

**Reply by kasgithub8** [reply]
That's tough, thanks for the response and the request tho!

**Reply by kasgithub8** [reply]
Hey @melohagan,

I just found out that the "markdown viewer" is perfect for links!
